src/ape/managers/chain.py

```diff
@@ -441,6 +444,11 @@ def get(
         contract_type = self._get_contract_type_from_disk(address)
 
         if not contract_type:
+            # Contract could be a minimal proxy
+            target = self.resolve_proxy(address)
+            if target:
+                return self.get(target)
```

Comment:
Just thought of this: Maybe this could happen later, but we should also be caching these as well and we may need to adjust our caching schema to include the proxy type.

Reply:
added `proxy_info` with some metadata which we can cache. i think we also need to add ttl or user-initiated cache invalidation.